crates/forge-runner/src/compiled_runnable.rs

```diff
@@ -26,8 +29,24 @@ pub struct ValidatedForkConfig {
     pub block_id: BlockId,
 }
 
-#[derive(Debug, Clone, PartialEq, Deserialize)]
-pub struct FuzzerConfig {
-    pub fuzzer_runs: u32,
-    pub fuzzer_seed: u64,
+impl TryInto<ValidatedForkConfig> for RawForkParams {
+    type Error = anyhow::Error;
+
+    fn try_into(self) -> Result<ValidatedForkConfig, Self::Error> {
+        let block_id = match self.block_id_type.to_lowercase().as_str() {
+            "number" => BlockId::Number(self.block_id_value.parse().unwrap()),
+            "hash" => {
+                BlockId::Hash(Felt252::from(self.block_id_value.parse::<BigInt>().unwrap()).into_())
+            }
+            "tag" => {
+                assert_eq!(self.block_id_value, "Latest");
+                BlockId::Tag(Latest)
+            }
+            _ => unreachable!(),
+        };
+        Ok(ValidatedForkConfig {
+            url: self.url.parse()?,
+            block_id,
+        })
+    }
```

Comment:
Why was this moved here? 🤔

Reply:
`RawForkParams` has been moved to the shared crate, we cannot implement a trait for it (as neither the trait nor the struct has been defined in this crate)

Reply:
This will be removed with #1049 that I'm working on rn anyways